gateway/README.md

```diff
@@ -11,149 +11,260 @@ This basic example demonstrates how to expose a service or a function through an
+- A JWT token fetched from the Console UI which is later referred to as **\{jwt-token\}**. For more details, see the **Fetch JWT token** section in the **Exposure through the console UI**.
+- If you run Kyma locally, add the `http-db-service.kyma.local` to your hosts file.
 
 #### Create a service
 
-> **NOTE:** Almost all steps in this tutorial refer to a lambda but you can apply them also to the service Deployment by changing the directory and names in some places.
-
 1. Export your Namespace as a variable. Replace the `{namespace}` placeholder in the following command and run it:
 
    ```bash
    export KYMA_EXAMPLE_NS="{namespace}"
    ```
+   
+2. Export your Kyma domain as a variable. Replace the `{domain}` placeholder in the following command and run it::
 
-2. Apply one of the `deployment.yaml` files from the `lambda` or `service` directory in this example.
+    ```bash
+   export KYMA_EXAMPLE_DOMAIN="{domain}"
+   ```
+
+3. Apply the `deployment.yaml` file from the `service` directory in this example.
 
    ```bash
-   kubectl apply -f ./lambda/deployment.yaml -n $KYMA_EXAMPLE_NS
+   kubectl apply -f ./service/deployment.yaml -n $KYMA_EXAMPLE_NS
    ```
 
 #### Expose a service without authentication
 
-Run this command:
+Run this command to create an APIRule:
 
 ```bash
-kubectl apply -f ./lambda/api-without-auth.yaml -n $KYMA_EXAMPLE_NS
+kubectl apply -f ./service/api-without-auth.yaml -n $KYMA_EXAMPLE_NS
 ```
 
-#### Test the APIs without authentication
+#### Test the APIRules without authentication
 
 To perform a test, use the following command:
 
 ```bash
-curl -ik https://hello.kyma.local
-# > 200 Hello world
+curl -ik https://http-db-service.$KYMA_EXAMPLE_DOMAIN/orders
+# > 200 []
+```
+
+#### Expose a service with JWT authentication
+
+> **NOTE:** If you followed the steps in **Expose a service without authentication** section, the previously created APIRule will be updated after applying the templates.
+
+The JWT authentication settings require to provide a list of trusted issuers. To create an APIRule with the JWT authentication settings, run: 
+
+```bash
+cat <<EOF | kubectl apply -f - 
+apiVersion: gateway.kyma-project.io/v1alpha1
+kind: APIRule
+metadata:
+  labels:
+    example: gateway-service
+  name: http-db-service
+  namespace: $KYMA_EXAMPLE_NS
+spec:
+  gateway: kyma-gateway.kyma-system.svc.cluster.local
+  rules:
+    - accessStrategies:
+        - config:
+            jwks_urls:
+              - http://dex-service.kyma-system.svc.cluster.local:5556/keys
+            trusted_issuers:
+              - https://dex.$KYMA_EXAMPLE_DOMAIN
+          handler: jwt
+      methods:
+        - GET
+        - POST
+        - PUT
+        - DELETE
+      path: /.*
+  service:
+    host: http-db-service.kyma.local
+    name: http-db-service
+    port: 8017
```

Comment:
Should include `EOF` in the last line of this yaml. Without it, the resource won't be created. Look at [this example (step 2)](https://kyma-project.io/docs/components/api-gateway/#tutorials-expose-and-secure-a-service-register-an-o-auth2-client-and-get-tokens)